app/helpers/spotlight/languages_helper.rb

```diff
@@ -19,5 +19,14 @@ def locale_selecter_dropown_options
 
       languages.reject { |language| language.locale.to_s.casecmp(I18n.locale.to_s).zero? }.sort_by(&:to_native)
     end
+
+    ##
+    # Can determine whether the current page is using the application's default
+    # locale
+    # @return [Boolean]
+    def default_language?
+      return true unless params[:locale]
+      params[:locale].to_s == I18n.default_locale.to_s
+    end
```

Comment:
Is this offering something better than what `I18n.locale == I18n.default_locale`?

I think that would allow us to avoid casting to string and we can remove the guard clause (because we'll always have a locale).

Reply:
Nope.. I guess I didn't realize that was the same thing

Reply:
My test doesn't pass now, even when passing in strings as params. 

hmm..